src/WikiParsing/AdjectiveWikiParsing.fs

```diff
@@ -0,0 +1,101 @@
+﻿module WikiParsing.AdjectiveWikiParsing
+
+open FSharp.Data
+
+open Common.WikiArticles
+open WikiParsing.ConcreteArticles
+
+type private WikiAdjectiveNový = HtmlProvider<"https://cs.wiktionary.org/wiki/nový">
+type private WikiAdjectiveStarý = HtmlProvider<"https://cs.wiktionary.org/wiki/starý">
+type private WikiAdjectiveVrchní = HtmlProvider<"https://cs.wiktionary.org/wiki/vrchní">
+
+type WikiDeclension = 
+    {
+        SingularNominative: string
+        SingularGenitive: string
+        SingularDative: string
+        SingularAccusative: string
+        SingularVocative: string
+        SingularLocative: string
+        SingularInstrumental: string
+        PluralNominative: string
+        PluralGenitive: string
+        PluralDative: string
+        PluralAccusative: string
+        PluralVocative: string
+        PluralLocative: string
+        PluralInstrumental: string
+    }
+
+type WikiComparison =
+    {
+        Positive: string
+        Comparative: string
+        Superlative: string
+    }
+
+let get1Declension (AdjectiveArticleWithPlural (AdjectiveArticle { Text = text })) =
+    let data = WikiAdjectiveNový.Parse text
+    {
+        SingularNominative =   data.Tables.``Skloňování[editovat]``.Rows.[0].``singulár - mužský životný``
+        SingularGenitive =     data.Tables.``Skloňování[editovat]``.Rows.[1].``singulár - mužský životný`` 
+        SingularDative =       data.Tables.``Skloňování[editovat]``.Rows.[2].``singulár - mužský životný`` 
+        SingularVocative =     data.Tables.``Skloňování[editovat]``.Rows.[4].``singulár - mužský životný`` 
+        SingularAccusative =   data.Tables.``Skloňování[editovat]``.Rows.[3].``singulár - mužský životný`` 
+        SingularLocative =     data.Tables.``Skloňování[editovat]``.Rows.[5].``singulár - mužský životný`` 
+        SingularInstrumental = data.Tables.``Skloňování[editovat]``.Rows.[6].``singulár - mužský životný`` 
+        PluralNominative =     data.Tables.``Skloňování[editovat]``.Rows.[0].``plurál - mužský životný``  
+        PluralGenitive =       data.Tables.``Skloňování[editovat]``.Rows.[1].``plurál - mužský životný``  
+        PluralDative =         data.Tables.``Skloňování[editovat]``.Rows.[2].``plurál - mužský životný``  
+        PluralAccusative =     data.Tables.``Skloňování[editovat]``.Rows.[3].``plurál - mužský životný``  
+        PluralVocative =       data.Tables.``Skloňování[editovat]``.Rows.[4].``plurál - mužský životný``  
+        PluralLocative =       data.Tables.``Skloňování[editovat]``.Rows.[5].``plurál - mužský životný``  
+        PluralInstrumental =   data.Tables.``Skloňování[editovat]``.Rows.[6].``plurál - mužský životný``  
+    }
+
+let get2Declension (AdjectiveArticleWithPlural (AdjectiveArticle { Text = text })) =
```

Comment:
What's get1, get2... mean? Is it different page structures?

Reply:
It's bad naming then :)

Well, the thing is that some adjectives can be also nouns (e.g. `vrchní`) and to get the actual adjective declension we need to parse the last declension table on the page. So it is a simple idea (a bit subtle also but well) - if the word can be an adjective then adjective declension will be in the last declension table on the page.

Compare [nový](https://cs.wiktionary.org/wiki/nov%C3%BD) vs [vrchní](https://cs.wiktionary.org/wiki/vrchn%C3%AD). Check if the new naming is any clearer...